WikiDrips/WikiDoc.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  WikiDoc.swift
+//  WikiDrips
+//
+//  Created by Bart Cullimore on 4/3/17.
+//  Copyright © 2017 Bart Cullimore. All rights reserved.
+//
+
+import Foundation
+
+public class WikiDoc
+{
+    public let title = "Test Title String"
+    public let pubDate = Date.init()
+    public var imageInitials: String {
+        get {
+            let nameArray = self.title.components(separatedBy: " ")
+            return "\(nameArray.first?.characters.first!)\(nameArray.last?.characters.first!)"
```

Comment:
Exclamation marks are evil. Look into breaking this up to multiple lines and correctly handling when values are nil.